new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -68,15 +68,17 @@ public static boolean upgradeOperator(OperatorParams params) {
   }
 
   /**
-   * Makes a REST call to the Operator to scale the domain.
+   * Scale the cluster of the domain in the specified namespace .
    *
-   * @param domainUid domainUid of the domain
-   * @param clusterName cluster in the domain to scale
-   * @param numOfServers number of servers to scale upto.
+   * @param domainUid domainUid of the domain to be scaled
+   * @param clusterName cluster in the domain to be scaled
+   * @param numOfServers number of servers to be scaled to.
    * @return true on success, false otherwise
+   * @throws ApiException if Kubernetes client API call fails
    */
-  public static boolean scaleDomain(String domainUid, String clusterName, int numOfServers) {
-    return Operator.scaleDomain(domainUid, clusterName, numOfServers);
+  public static boolean scaleCluster(String domainUid, String namespace, String clusterName, int numOfServers)
```

Comment:
can you move this method to domain actions section

Reply:
fixed